app/build.gradle

```diff
@@ -2,8 +2,6 @@ apply plugin: 'com.android.application'
 
 apply plugin: 'kotlin-android'
 
-apply plugin: 'kotlin-android-extensions'
```

Comment:
just curious - is this no longer needed because of the changes in this PR, or was it something that was unused from before?

Reply:
There is bug https://identitydivision.visualstudio.com/Engineering/_workitems/edit/2883053 created for this dependency and I include the fixing into this PR without a separate one. Will update the PR summary.